regexes/device/mobiles.yml

```
@@ -30,7 +30,7 @@ Ace:
 
 # SFR or SFR Mobile
 SFR:
-  regex: 'StarShine|StarTrail|STARADDICT|StarText|StarNaute|StarXtrem|StarTab|(?:Android 8.*; (?:AURA (?:PLUS|ULTRA)|AURA))(?: Build|[;)])'
+  regex: 'StarShine|StarTrail|STARADDICT|StarText|StarNaute|StarXtrem|StarTab|(?:Android 8(?:[\d.]*);(?: [\w-]+;)? (?:AURA (?:PLUS|ULTRA)|AURA))(?: Build|[;)])'
```

If you want to match a dot, you need to escape it. Otherwise it's a placeholder for any random character.
```suggestion
  regex: 'StarShine|StarTrail|STARADDICT|StarText|StarNaute|StarXtrem|StarTab|(?:Android 8(?:[\d\.]*);(?: [\w-]+;)? (?:AURA (?:PLUS|ULTRA)|AURA))(?: Build|[;)])'
```
same might apply to the other changes.

---

there is no need to escape the dot in square brackets, this is not any character.

here is a demo

---

You're right. Should have taken a closer look and think about it, instead of trying to do a quick review while doing something else 🙈